### DISCUSSION TITLE
strange streaming at bottom

### DISCUSSION THREAD

**Discussion by waleedd32** [k-yle/rtsp-relay#148] [Q&A]
Everything seems working to me except i am getting strange picture at bottom of live streaming.

When i use vlc media player to watch live stream it looks perfect, but when using rtsp-relay to stream to my web i am getting this strange thing at bottom of video, any idea how to fix that ?

![image](https://user-images.githubusercontent.com/32420369/154864219-f5b958a8-d028-46e1-a5b5-24c9d3604e93.png)
and i am using react example of yours:

import React, { useRef, useEffect } from 'react';
import ReactDOM from 'react-dom';
import { loadPlayer } from 'rtsp-relay/browser';

function App() {
const canvas = useRef(null);

useEffect(() => {
if (!canvas.current) throw new Error('Ref is null');

loadPlayer({
  url: 'ws://localhost:2000/api/stream',
  canvas: canvas.current,
});
}, []);

return (
< div>
{' '}
< div style = { { border: '5px solid red ' }}>
< canvas ref ={ canvas} />
< / div>
< / div>
);
}

export default App;

**Comment by k-yle** [comment]
see https://github.com/k-yle/rtsp-relay#improving-the-video-quality
